examples/hello_part/src/main.rs

```diff
@@ -41,6 +44,10 @@ impl Partition<Hypervisor> for Hello {
                 .create_sampling_port_source(Name::from_str("Hello").unwrap())
                 .unwrap();
             SOURCE_HELLO.set(source).unwrap();
+            let socks = Hypervisor::get_udp_sockets();
+            let sock = socks.first().unwrap();
+            sock.connect("127.0.0.1:34256").unwrap();
```

Comment:
Why? UDP is a connection-less protocol?

Reply:
Because this is the port the hello messages should be received on. We could also use send_to instead, but since the destination does not change in this example, what would be the point?

Reply:
> Because this is the port the hello messages should be received on. We could also use send_to instead, but since the destination does not change in this example, what would be the point?

Because `sendto(2)` and it's receiving counter-part are, to my knowledge, simply more commonly used for this purpose:
- [OpenBSD's ntpd(8)](https://github.com/openbsd/src/blob/fad6d19425b1a957794f23d76396489614b81eac/usr.sbin/ntpd/ntp_msg.c#L53)
- [libcurl's tftp](https://github.com/curl/curl/blob/3ef31a16fcab6f914a71fc29bef950c544241342/lib/tftp.c#L531)

I'm neutral on your design choice though, I just wanted to point out the semantic "irregularity" with this approach.